ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c

```diff
@@ -268,6 +269,7 @@ _tflite_ssd_loadBoxPrior (bounding_boxes * bdata)
     prev_reg = registered;
   }
 
+error:
```

Comment:
It seems return with success could come to this line.
I think it'd better change label to `done` or equivalent.